homeassistant/components/camera/zoneminder.py

```diff
@@ -107,7 +107,7 @@ def update(self):
                             self._monitor_id)
             return
 
-        if status_response['success'] is False:
+        if 'success' in status_response:
```

Comment:
I'm not that familiar with the requests library being used but I think this is inverting the intention of the condition.